arber-salihu/loGIStics
# Add main elements to side panel without major functionality

== Test Steps ==
1. Run `npm install`, then `npm run dev` and go local deployment 
2. Check the components in the side menu
 - They should resemble the layout in the wireframe
 - There is currently no functionality, but there is some mock data for both alerts and boundaries
 - Things you do in one tab (alerts or boundaries) will not be persisted when you come back to that tab bc the component is destroyed when the tab is switched. This will be updated in the future.
3. Review the code changes
 - For the structure of the side menu components, I created a side-menu component, which contains two components for the 'info' view, and the 'form' view. Let me know if this file structure should be updated.
